solutions/public/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>PG workshop - get users from the database</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <link rel="stylesheet" href="main.css">
+  </head>
+  <body>
+    <nav>
+      <div class="nav-wrapper">
+        <a href="#" class="brand-logo">Users</a>
+      </div>
+    </nav>
+    <!-- <form class="form" action="/create-user" method="post">
+      <input type="text" name="name" value="" placeholder="name">
+      <input type="text" name="location" value="" placeholder="location">
+      <button type="submit" name="button">+ Add user</button>
+    </form> -->
```

Comment:
is this still in the solution, because I can't see anything related. if it's not please delete it